### DISCUSSION TITLE
determine which tag to use

### DISCUSSION THREAD

**Discussion by Nathan187** [semantic-release/semantic-release#2278] [Configuration] [answered]
i inherited an app from someone who left little documentation when it comes to using this.

i'm trying to figure out, when i try to get this to work, i get a message:

Found git tag v3.22.0 associated with version 3.22.0 on branch master

well, that isn't the latest tag.  is there a specific way to tell it which tag to use (or something like that?)

**Comment by travi** [comment, marked as answer]
you should be able to find the information you need in https://semantic-release.gitbook.io/semantic-release/usage/configuration#existing-version-tags

**Reply by Nathan187** [reply]
this really helped man.  thanks a lot (no sarcasm)
